hw/ip/lc_ctrl/data/lc_ctrl.hjson

```diff
@@ -77,7 +77,13 @@
       randcount: "128",
       randtype:  "data",
     }
-    { name:      "RndCnstLcKeymgrDivTestDevRma",
+    { name:      "RndCnstLcKeymgrDivTestUnlocked",
+      desc:      "Compile-time random bits for lc state group diversification value",
```

Comment:
Nit: it's intuitive from the name, but can probably also add the name of the `lc state ` in the description of each of these.

Reply:
Yes good point, let me make those descriptions a bit more informative.